### PR TITLE
fix(query-engine): Rename N-API to Node-API and link to it from preview features page

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ The following Preview feature flags are available for Prisma Client:
 * [`orderByRelations`](../prisma-client/filtering-and-sorting#sort-by-relation-aggregate-preview)
 * [`selectRelationCount`](../prisma-client/aggregation-grouping-summarizing#count-relations)
 * [`orderByAggregateGroup`](../prisma-client/aggregation-grouping-summarizing#order-by-aggregate-group-preview)
-* `nApi` (N-API support)
+* [`nApi` (Node API support)](../prisma-engines/query-engine#enable-the-node-api-n-api-preview)
```
Comment (janpio) [inline content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx]: This link needs to be double checked.

**Comment by mhwelander** [comment]
@janpio tested this and link works fine - is this ready to merge or are we waiting for a rename announcement?

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx (hunk 1 above)
Comment (mhwelander) [reply content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx]: Checked in deploy preview, all good

File: content/200-concepts/100-components/08-prisma-engines/200-query-engine.mdx [hunk 2]
```diff
@@ -80,9 +80,9 @@ const prisma = new PrismaClient({
 
 Learn more in the [Debugging](../prisma-client/debugging) and [Logging](../prisma-client/working-with-prismaclient/logging) pages of the docs.
 
-## Enable the N-API Preview
+## Enable the Node API (N-API) Preview
 
-N-API is available in [2.20.0](https://github.com/prisma/prisma/releases/2.20.0) and later. It is a new technique for binding Prisma's Rust-based query engine directly to Prisma Client. This reduces the communication overhead between the Node.js and Rust layers when resolving Prisma Client's database queries.
+Node API is available in [2.20.0](https://github.com/prisma/prisma/releases/2.20.0) and later. It is a new technique for binding Prisma's Rust-based query engine directly to Prisma Client. This reduces the communication overhead between the Node.js and Rust layers when resolving Prisma Client's database queries.
```
Comment (janpio) [inline content/200-concepts/100-components/08-prisma-engines/200-query-engine.mdx]:
```suggestion
Node-API is available in [2.20.0](https://github.com/prisma/prisma/releases/2.20.0) and later. It is a new technique for binding Prisma's Rust-based query engine directly to Prisma Client. This reduces the communication overhead between the Node.js and Rust layers when resolving Prisma Client's database queries.
```

File: content/200-concepts/100-components/08-prisma-engines/200-query-engine.mdx [hunk 3]
```diff
@@ -80,9 +80,9 @@ const prisma = new PrismaClient({
 
 Learn more in the [Debugging](../prisma-client/debugging) and [Logging](../prisma-client/working-with-prismaclient/logging) pages of the docs.
 
-## Enable the N-API Preview
+## Enable the Node API (N-API) Preview
```
Comment (janpio) [inline content/200-concepts/100-components/08-prisma-engines/200-query-engine.mdx]:
```suggestion
## Enable the Node-API (N-API) Preview
```

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx (hunk 1 above)
Comment (janpio) [reply content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx]:
```suggestion
* [`nApi` (Node-API support)](../prisma-engines/query-engine#enable-the-node-api-n-api-preview)
```

File: content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx [hunk 4]
```diff
@@ -10,7 +10,7 @@ The following [Preview](../../../../about/releases#preview) feature flags are av
 * [`orderByRelations`](../prisma-client/filtering-and-sorting#sort-by-relation-aggregate-value-preview) [Submit feedback](https://github.com/prisma/prisma/issues/5438)
 * [`selectRelationCount`](../prisma-client/aggregation-grouping-summarizing#count-relations) [Submit feedback](https://github.com/prisma/prisma/issues/6312)
 * [`orderByAggregateGroup`](../prisma-client/aggregation-grouping-summarizing#order-by-aggregate-group-preview) [Submit feedback](https://github.com/prisma/prisma/issues/6545)
-* `nApi` (N-API support) [Submit feedback](https://github.com/prisma/prisma/issues/6301)
+* [`nApi` (Node-API support)](../prisma-engines/query-engine#enable-the-node-api-n-api-preview) [Submit feedback](https://github.com/prisma/prisma/issues/6301)
 * [filterJson](../prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field) [Submit feedback](https://github.com/prisma/prisma/issues/7135)
```
Comment (janpio) [inline content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx]:
```suggestion
* [`filterJson`](../prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field) [Submit feedback](https://github.com/prisma/prisma/issues/7135)
```

**Comment by janpio** [comment]
We decided to go with `Node-API` but that should not be fixed, so ready to merge.

**Pull request merged**